Cakefile

```diff
@@ -85,7 +85,16 @@ buildJsInLocales = ->
         name     = file.replace '.json', '.js'
         fs.writeFileSync "./build/client/app/locales/#{name}", exported
         # add locales at the end of app.js
+    for file in fs.readdirSync './server/locales/'
```

Comment:
In node, you can directly `locales = require('./somefile.json')` , why do you convert them to JS ?